capnp-rpc-lwt/capnp_rpc_lwt.mli

```diff
@@ -203,8 +203,34 @@ module Restorer : sig
         This may be useful for interoperability with non-secure clients that expect
         to use a plain-text service ID (e.g. "calculator"). It could also be
         useful if [name] is some unguessable token you have generated yourself. *)
+
+    val digest : Auth.hash -> t -> string
```

Comment:
Is the precise hash function used defined by capnp or is there a choice made by the OCaml implementation here? Might be worth documenting it in case of future weaknesses with the one chosen.

Reply:
The hash algorithm to use is an argument to the function. The protocol doesn't care what it is. Remote peers never see the hash - it's just a local security measure. I've added a note in `Table.of_loader` about this.